pkg/agent/proxy/proxier.go

```diff
@@ -476,19 +494,19 @@ func (p *proxier) installServices() {
 					// of the Service are different, install two groups. One group has all Endpoints, the other has only
 					// local Endpoints.
 					groupID := p.groupCounter.AllocateIfNotExist(svcPortName, true)
-					if err = p.ofClient.InstallServiceGroup(groupID, affinityTimeout != 0, localEndpointUpdateList); err != nil {
+					if err = p.ofClient.InstallServiceGroup(groupID, affinityTimeout != 0, nil, localEndpointUpdateList); err != nil {
```

Comment:
Better to add a comment here to explain that a Service in multi-cluster scenario would not use NodePort/LoadBalancer type.

Reply:
Done.